lib/twitter_cldr/resources/collation_tries_importer.rb

```diff
@@ -34,7 +34,7 @@ class CollationTriesImporter < Importer
         '004C | 0387; [, DB A9, 05]' => '004C 0387; [3D, 05, 8F][, DB A9, 05]',
       }
 
-      requirement :cldr, '21'
+      requirement :cldr, '35'
```

Comment:
Unfortunately our collation implementation isn't compatible with the latest versions of CLDR and ICU. I wish it were as easy as changing this version number, but sadly we're stuck with CLDR v21 for collation for now.